BUILD

```diff
@@ -29,19 +27,15 @@ cc_library(
     ],
 )
 
-load("@rules_python//python:packaging.bzl", "py_wheel")
-
-load(":package.bzl", "py_package")
+load("@rules_python//python:packaging.bzl", "py_package", "py_wheel")
 
 py_package(
     name = "enzyme_jax_data",
     # Only include these Python packages.
-    packages = ["@//enzyme_jax:enzyme_call.so", "@llvm-project//clang:builtin_headers_gen"],
```

Comment:
Warning: I don't understand what `packages` is supposed to do. If I use the line as is, the `.so` file is *not* included in the wheel...

Reply:
So it's supposed to force the so (And those headers) to be made available in the pip package.

Reply:
It does look like CI thinks that it can no longer find them, so we may need something else here (it worked in the past so I'm not sure what would've changed, and I'm curious if putting it back fixes anything)

Reply:
When it was there, CI broke. I could observe the same behavior locally: only the py files ended up in the wheel. I don't know what changed, though...

Reply:
Ah yeah so the problem is that the `package.bzl` forces the dependencies to be included as well, so by getting rid of our version of package.bzl you've removed the ability to import the actual dependencies.

Revert back to our package.bzl and see if it works?

Reply:
I have also tried that. I started looking into the wheel rule because it broke, even before updating rules_python. Also, the old package.bzl doesn't have filtering logic for all I can tell.

Is there anything in the current behavior that is undesirable?